manifests/charts/base/files/profile-platform-openshift.yaml

```diff
@@ -17,3 +17,4 @@ seLinuxOptions:
   type: spc_t
 # Openshift requires privileged pods to run in kube-system
 trustedZtunnelNamespace: "kube-system"
+platform: "openshift"
```

Comment:
nit: please add a newline

Reply:
Hello, jacob. I updated my PR and appended a newline there.

Do you know a way of building an `istioctl` binary from my PR local branch and then I can test it ? Thanks